api/app/src/routes/medical/patient.ts

```diff
@@ -39,6 +43,16 @@ router.post(
     const facilityId = getFromQueryOrFail("facilityId", req);
     const payload = patientCreateSchema.parse(req.body);
 
+    const patientExists = await getPatientByDemo({
+      facilityId,
+      cxId,
+      demo: payload,
+    });
+
+    if (patientExists) {
+      return res.status(status.OK).json(dtoFromModel(patientExists));
+    }
```

Comment:
I suggest we have this as part of `createPatient` command:
- it concentrates logic in one place, leaving route's responsibility to deal with HTTP/API related stuff and conversions to/from the domain
- it helps us later on when we move to generate routes based on api specs/docs

Reply:
The reason for putting it here is because we dont want to create the patient in CW either

Reply:
No problem, we could have `createPatient` to return something to indicate whether it was created or not.

Actually, I wanted to move the call to CW to the command, it could be a good opportunity to do so.

If you agree but prefer to do it later, could just create a ticket so we can get this PR out.